packages/app/src/app/pages/Profile/Showcase/SandboxInfo/elements.ts

```diff
@@ -18,7 +19,6 @@ export const Container = styled.div`
 `;
 
 export const Title = styled.h1`
-  font-weight: 400;
```

Comment:
This one wasn't needed?

Reply:
A bit further down we have

```css
font-weight: 300;
```

so this one's obsolete